<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employe2</class>
 <widget class="QWidget" name="employe2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>931</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>410</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	
	qproperty-icon: url(:/Downloads/trier.png);
	qproperty-iconSize: 30px 30 px;
text-align:center;
	color: rgb(27, 75, 101);
}</string>
      </property>
      <property name="text">
       <string>Trier par</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtontri">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxtri">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>1051</width>
       <height>271</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_49">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>Liste des employees</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf_button">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	border-image: url(:/2A3/qt/tsawer/pdf (2).png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="tab_employe">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>581</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>321</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton #pushButton{
	
	background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton #pushButton:hover{
	
	background-color: rgb(98, 185, 203);
}
QPushButton #pushButtonp:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(98, 185, 203);
}</string>
     </property>
     <widget class="QLabel" name="label_62">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>281</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_63">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(27, 75, 101);
</string>
      </property>
      <property name="text">
       <string>Formulaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_64">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_65">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_67">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_68">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>Salaire</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_cin">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_salaire">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}

</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_69">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(27, 75, 101);</string>
      </property>
      <property name="text">
       <string>Metier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_metier">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_modifier">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}

</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_SUPP">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>330</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="login2">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stat">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>stat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>330</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Detection:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>330</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>off</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
